module.d.ts

```diff
@@ -98,6 +98,17 @@ declare module '@solana/web3.js' {
     }>;
   };
 
+  export type ConfirmedTransaction = {
+    slot: number;
+    transaction: Transaction;
+    meta: {
```

Comment:
Worth creating a type for this, since it's a dupe of meta in ConfirmedBlock?

Reply:
@CriesofCarrots good call, done!